scripts/globals/mannequins.lua

```diff
@@ -2,7 +2,6 @@
 -- Information on Mannequins
 ----------------------------
 
-require("scripts/globals/items")
```

Comment:
need to kill one of the line returns here

Reply:
I THINK I just took care of all style changes needed after the mass deletion.We'll see what the CI says XD